src/web/topic/components/TopicWorkspace/TopicWorkspace.tsx

```diff
@@ -27,13 +29,15 @@ export const TopicWorkspace = () => {
           flexDirection: isLandscape ? "row" : "column-reverse",
         }}
       >
-        <TopicPane isLandscape={isLandscape} />
-
+        <TopicPane
+          anchor={isLandscape ? "left" : "bottom"}
+          tabs={isAlternativeLayout ? ["Details"] : ["Details", "Views"]}
+        />
         <Box height="100%" flex="1" overflow="auto">
           {format === "table" && <CriteriaTable />}
           {format === "diagram" && <Diagram />}
         </Box>
-
+        {isAlternativeLayout && <TopicPane anchor="right" tabs={["Views"]} />}
```

Comment:
```suggestion
        {isAlternativeLayout && <TopicPane anchor="right" tabs={["Details"]} />}
```

Bump from this comment https://github.com/amelioro/ameliorate/pull/442#discussion_r1640157736

Reply:
got it, had it in my head flipped for w/e reason